Formula/kubetail.rb

```diff
@@ -0,0 +1,24 @@
+class Kubetail < Formula
+  desc "Bash script to tail Kubernetes logs from multiple pods at the same time"
+  homepage "https://github.com/johanhaleby/kubetail"
+  url "https://github.com/johanhaleby/kubetail/archive/1.6.13.tar.gz"
+  sha256 "a4aeea1cffede44d5e8b030b6ead3ffe25fd3c1e7c5b7c82905df65cceac1254"
+  license "Apache-2.0"
+  head "https://github.com/johanhaleby/kubetail.git", branch: "master"
+
+  depends_on "kubernetes-cli" => :test
```

Comment:
How is it only in the test and not at runtime?

Reply:
My thinking is this is the CLI, and users can install the k8s client in their preferred way.

Reply:
Will it work without a k8 client?